libs/common-services/src/main/java/uk/gov/di/ipv/core/library/service/YamlConfigService.java

```diff
@@ -19,6 +20,20 @@ public class YamlConfigService extends ConfigService {
     private static final String PATH_SEPARATOR = "/";
     private static final String CORE = "core";
     private static final String FEATURE_SETS = "features";
+    private final ThreadLocal<List<String>> featureSet = new ThreadLocal<>();
+
+    @Override
+    public List<String> getFeatureSet() {
+        return featureSet.get();
+    }
+
+    public void setFeatureSet(List<String> featureSet) {
```

Comment:
does this need an `@Override` too?

Reply:
Neither should need it as they're defined as `abstract` in `ConfigService`, will remove - thank you for the spot